changelog/unreleased/pull-4480

```diff
@@ -0,0 +1,9 @@
+Enhancement: Allow setting REST password and username via environment variables
+
+Previously, it was only possible to specify the REST server username and
+password in the repository URL. This meant it was not possible to use
+authentication in contexts where the repository URL is public. Restic now allows
```

Comment:
There is already the option to use `--repository-file` to hide the URL. The environment variables just add another way to achieve that goal.

Reply:
Changed it to clarify what I meant, still not sure if the current version is better.

Reply:
The changed text is better, thanks.